### PR TITLE
EIP 2400 - Tx hash URI: Move from Stagnant to Review

### PR DESCRIPTION
**Pull request by 3esmit**
As decided on https://github.com/ethereum-cat-herders/EIPIP/issues/149 to move it to Review process.

### CODE REVIEW

File: EIPS/eip-2400.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+status: Review
+created: 2019-11-05
+requires: 155, 831
+---
+
+## Simple Summary
+
+A standardized URI for transaction receipt with complete information about transaction.
+
+A standard way of representing a submitted transaction.
```
Comment (Pandapip1) [inline EIPS/eip-2400.md:17]: No longer a valid EIP section. Convert this to a description.

File: EIPS/eip-2400.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+### Use-cases 
+
+Transaction Receipt URIs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications. A standardized URI format allows for instant invocation of the user’s preferred transaction explorer application. Such as:
+
+- In web3 (dapps, mining pools, exchanges), links would automatically open user's preferred transaction explorer; 
+- In wallets, for users sharing transaction receipts easier; 
+- In chat applications, as a reply to an [ERC-681] transaction request;
+- In crypto vending machines, a QRCode can be displayed when transactions are submitted;
+- Anywhere transaction receipts are presented to users.
```
Comment (Pandapip1) [inline EIPS/eip-2400.md:32]: Should be a sub-section of the motivation.

File: EIPS/eip-2400.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+#### Examples
+
+##### [Simple ETH transfer](https://etherscan.io/tx/0x1143b5e38fe3cf585fb026fb9b5ce35c85a691786397dc8a23a07a62796d8172): 
+`ethereum:tx-0x1143b5e38fe3cf585fb026fb9b5ce35c85a691786397dc8a23a07a62796d8172@1`  
+
+##### [ERC20 transfer](https://etherscan.io/tx/0x5375e805b0c6afa20daab8d37352bf09a533efb03129ba56dee869e2ce4f2f92):
+
+`ethereum:tx-0x5375e805b0c6afa20daab8d37352bf09a533efb03129ba56dee869e2ce4f2f92@1?method="transfer(address,uint256)"&events="Transfer(!address,!address,uint256)"` 
+
+##### [Complex contract transaction](https://etherscan.io/tx/0x4465e7cce3c784f264301bfe26fc17609855305213ec74c716c7561154b76fec#eventlog):
```
Comment (Pandapip1) [inline EIPS/eip-2400.md:83]: No external links

File: EIPS/eip-2400.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+Interoperability for web3: wallets, browsers and messengers.
+
+## Specification
+
+### Syntax
+
+Transaction receipt URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    receipt                 = erc831_part transaction_hash [ "@" chain_id ] [ "?" parameters ]
+    erc831_part             = "ethereum:tx-"
```
Comment (Pandapip1) [inline EIPS/eip-2400.md:45]: EIP-831 is stagnant and should not be referenced.

I suggest choosing a different schema. Perhaps `ethereum-tx`?

File: EIPS/eip-2400.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+## References
+    
+* [EIP-155]
+* [ERC-831]
+* [ERC-681]
+
+## Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (Pandapip1) [inline EIPS/eip-2400.md:106]:
```suggestion
Copyright and related rights waived via [CC0](../LICENSE.md).
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

File: EIPS/eip-2400.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
+---
+eip: 2400
```
Comment (Pandapip1) [inline EIPS/eip-2400.md:2]:
```suggestion
eip: 5217
```

Comment (Pandapip1) [reply EIPS/eip-2400.md:2]: Never mind, disregard this. For some reason, GitHub is showing this as a brand-new file.

**Comment by Pandapip1** [comment]
This currently has merge conflicts. Would you mind resolving them?

File: EIPS/eip-2400.md (hunk 4 above)
Comment (3esmit) [reply EIPS/eip-2400.md:45]: Perhaps just dont mention 831 and keep it as it is.

Comment (Pandapip1) [reply EIPS/eip-2400.md:45]: If you want, you could submit your own PR to move it out of stagnant (and add yourself as an author), and if ligi doesn't respond, either micah or lightclient could override the author approval requirement.

File: EIPS/eip-2400.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+title: URL Format for Transaction Receipts
```
[the remark this answers is not in the document]
Comment (3esmit) [reply EIPS/eip-2400.md]: I didnt understood why you commented this on line 3, but I guess you mean Description at the body of EIP. Than, I resolve this.

File: EIPS/eip-2400.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+- In web3 (dapps, mining pools, exchanges), links would automatically open user's preferred transaction explorer; 
+- In wallets, for users sharing transaction receipts easier; 
+- In chat applications, as a reply to an [ERC-681] transaction request;
+- In crypto vending machines, a QRCode can be displayed when transactions are submitted;
+- Anywhere transaction receipts are presented to users.
+
+## Motivation
+
+Interoperability for web3: wallets, browsers and messengers.
```
[the remark this answers is not in the document]
Comment (3esmit) [reply EIPS/eip-2400.md]: I hope now it's more clear!

**Comment by 3esmit** [comment]
I have worked on all suggested changes by @Pandapip1 .

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**